features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/RoomMemberListStateProvider.kt

```diff
@@ -17,27 +17,85 @@
 package io.element.android.features.roomdetails.impl.members
 
 import androidx.compose.ui.tooling.preview.PreviewParameterProvider
-import io.element.android.features.userlist.api.UserSearchResultState
-import io.element.android.features.userlist.api.aUserListState
 import io.element.android.libraries.architecture.Async
-import io.element.android.libraries.matrix.api.user.MatrixUser
-import io.element.android.libraries.matrix.ui.components.aMatrixUser
-import kotlinx.collections.immutable.ImmutableList
+import io.element.android.libraries.matrix.api.core.UserId
+import io.element.android.libraries.matrix.api.room.RoomMember
+import io.element.android.libraries.matrix.api.room.RoomMembershipState
 import kotlinx.collections.immutable.persistentListOf
+import kotlinx.collections.immutable.toImmutableList
 
 internal class RoomMemberListStateProvider : PreviewParameterProvider<RoomMemberListState> {
     override val values: Sequence<RoomMemberListState>
         get() = sequenceOf(
-            aRoomMemberListState(allUsers = Async.Success(persistentListOf(aMatrixUser()))),
-            aRoomMemberListState(allUsers = Async.Loading())
+            aRoomMemberListState(
+                roomMembers = Async.Success(
+                    RoomMembers(
+                        invited = persistentListOf(aRoomMember()),
```

Comment:
Nitpick: maybe you could customize one of the member's names so they don't look the same in the screenshots?